data/seeds/virtualParent.js

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('VirtualParent').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('VirtualParent').insert([
+        {id: 1, email: '', first_name:"Anne",last_name:"McVirtual"},
+        {id: 2, email: '', first_name:"Randy",last_name:"McVirtual"},
+        {id: 3, email: '', first_name:"Bob",last_name:"McVirtual"}
```

Comment:
Same thing here as with the virtual child. This is going to be inserted into the Parents table, so we need these properties to match. We will only have one virtual parent for these three children to belong to. The id here will be hard-coded, but we need it to be a value that doesn't currently exist. When you get to this point, let me know and I will help you find the right ID

Reply:
Will make adjustments tomorrow, thanks